akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/ReplayingEvents.scala

```diff
@@ -173,12 +173,12 @@ private[akka] final class ReplayingEvents[C, E, S](
 
     val sequenceNr = state.seqNr
     val msg = event match {
+      case Some(_: Message) | None =>
+        s"Exception during recovery. Last known sequence number [$sequenceNr]. " +
+        s"PersistenceId [${setup.persistenceId.id}]. ${cause.getMessage}"
       case Some(evt) =>
         s"Exception during recovery while handling [${evt.getClass.getName}] with sequence number [$sequenceNr]. " +
         s"PersistenceId [${setup.persistenceId.id}]. ${cause.getMessage}"
-      case None =>
-        s"Exception during recovery. Last known sequence number [$sequenceNr]. " +
-        s"PersistenceId [${setup.persistenceId.id}]. ${cause.getMessage}"
     }
```

Comment:
How's the compiler feeling about this, does it need something like `case _ =>` handling with the `None` removed?

Reply:
None is in the top case